filebeat/module/elasticsearch/gc/config/gc.yml

```diff
@@ -0,0 +1,15 @@
+type: log
+paths:
+{{ range $i, $path := .paths }}
+ - {{$path}}
+{{ end }}
+exclude_files: [".gz$"]
+multiline:
+  pattern: '^[0-9]{4}-[0-9]{2}-[0-9]{2}'
+  negate: true
+  match: after
+
+  fields:
```

Comment:
Here the indentation seems to be off `fields` and `fields_under_root` is probably not applying because of this.